composer.json

```diff
@@ -28,6 +28,7 @@
     }
   ],
   "minimum-stability": "dev",
+  "prefer-stable": true,
```

Comment:
> The "minimum-stability": "dev" setting is defined on composer.json file and it will install unstable dependencies during composer install work.
To install the stable dependencies, adding the "prefer-stable": true to resolve this problem.

Unstable dependency is not a problem, especially as I also maintain **nesbot/carbon** and participate in **spatie/period**, so if a change in one or the other would break **kylekatarnls/enhanced-period**, I would have no issue to fix it, and I want to be warned as soon as possible if it happen so `"prefer-stable": true` will do more harm than good.

Note that `minimum-stability` and `prefer-stable` does not affect users, only the root composer.json is took into account for those settings.

OK for the other change.